travis/run_on_non_pull_requests.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash -xe
+
+openssl aes-256-cbc -K $encrypted_8bf0fadbb437_key -iv $encrypted_8bf0fadbb437_iv -in deploy-key.enc -out deploy-key -d
```

Comment:
I assume the point of factoring this out is that we'll soon move this to a git sub-module?

If so, I think the environment variable name is per-repo: might want to separate this particular line out of the script?